src/models/User.ts

```diff
@@ -1,28 +1,32 @@
+import {
+  Grade, Role, ScholarshipType, Gender,
+} from './enums';
+
 interface UserProps {
-    age: number;
-    schoolYear: number;
-    role: string;
-    gender: string;
-    scholarshipPreferences: {
-        type: string,
-        major: string,
-        amount: number, 
-        school: string,
-        grade: number,
-        location: string
-    };
-    otherSettings: {
-        notifications: boolean,
-        deadlineNotifications: boolean,
-        relatedNotifications: boolean
-    }
+  age: number;
+  schoolYear: Grade;
+  role: Role;
+  gender: Gender;
+  scholarshipPreferences: {
+    type: ScholarshipType,
+    major: string,
+    amount: number,
+    school: string,
+    schoolYear: Grade,
+    location: string
+  };
```

Comment:
I would create a separate interface type for this and the `otherSettings` type

Reply:
Having them in the same file would be ok, right?